tests/test_smoke.py

```diff
@@ -796,11 +797,14 @@ def test_using_file_mounts_with_env_vars(generic_cloud: str):
 @pytest.mark.aws
 def test_aws_storage_mounts():
     name = _get_cluster_name()
-    storage_name = f'sky-test-{int(time.time())}'
+    random_name = int(time.time())
```

Comment:
nit: variable name: adding an integer to a 'name' is a little bit strange.

Reply:
updated to `timestamp = int(time.time())`